pages/about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Super SportCar Global Race</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="../Assets/sass/about.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
+</head>
+<body class="bg-light" id="body">
+  <div class="container">
```

Comment:
- [ ] On the `about` page, you're using margin on sides but no on the `Home` page, I invite you to remove the margin on about. 🌵 
- [ ] I think for the same reason, the burger icon appears bigger on the `about` page, and the size should be the same for both pages, please fix it. 🦊 
![image](https://user-images.githubusercontent.com/20563252/124698598-cf634f80-deae-11eb-8603-5a70121e6202.png)